src/lib.rs

```diff
@@ -89,6 +89,10 @@ pub use indexmap::{
 pub use indexset::{FnvIndexSet, IndexSet, Iter as IndexSetIter};
 pub use linear_map::LinearMap;
 pub use string::String;
+
+// Workaround https://github.com/rust-lang/rust/issues/119015. This is required so that the methods on `VecView` and `Vec` are properly documented.
+// cfg(doc) prevents `VecInner` being part of the public API.
```

Comment:
Is `cfg(doc)` actually necessary? I don't think you could even import `heapless::_` if you wanted to anyways.

Reply:
I don't think it is but I'm not certain. I trust the behaviour of `cfg` better than that of `_`.